benchmarks/benchmark2.ipynb

```diff
@@ -1594,66 +1579,6 @@
    "source": [
     "Results from this benchmark problem are displayed on the [simulation result page]({{ site.baseurl }}/simulations) for different codes."
    ]
-  },
```

Comment:
Removing the Disqus clutter from the footer is good, but having it there did give the impression we value feedback. Please add some language encouraging interaction, such as:
> Questions or comments? Click on the "Open Chat" button in the bottom-right to join the CHiMaD Phase Field community conversation.